### PR TITLE
Update message forwarder to take in Conductor workflow name

### PR DESCRIPTION
**Pull request by chubberlisk**
## Context

We're looking to set up a message forwarder for Phase 2 resubmissions, however atm it's hardcoded to start a Phase 1 workflow.

## Changes proposed in this PR

- Update message forwarder to take in a Conductor workflow name via an environment variable called `CONDUCTOR_WORKFLOW`.
  - By default, this is set to `bichard_phase_1`.
  - We do a check for valid workflows like destination type.

https://dsdmoj.atlassian.net/browse/BICAWS7-2994

### CODE REVIEW

File: packages/message-forwarder/src/forwardMessage/forwardMessage.ts [hunk 1]
```diff
@@ -21,6 +23,11 @@ const forwardMessage = async (
     return new Error(`Unsupported destination type: "${destinationType}"`)
   }
 
+  const conductorWorkflow = process.env.CONDUCTOR_WORKFLOW ?? "bichard_phase_1"
+  if (!conductorWorkflows.includes(conductorWorkflow)) {
+    return new Error(`Unsupported Conductor workflow: "${conductorWorkflow}"`)
+  }
+
   const aho = parseAhoXml(message)
```
Comment (emadkaramad) [inline packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: Should we parse both Aho and PncUpdateDataset?

Comment (bjpirt) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: Good point

Comment (bjpirt) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: I think this is just so we can extract the correlation ID for logging, maybe we should pull out the correlation ID extraction into its own function?

Comment (chubberlisk) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: Oooooooo yeahhh, that makes sense - alright if I do that in a separate PR?

Comment (emadkaramad) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: What about line 54?

return startBichardProcess(conductorWorkflow, aho, correlationId, conductorClient)
Does it start the new workflow with an AHO as input?

Comment (emadkaramad) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: It's safe to merge this PR. I just want to make sure when we use this for Phase 2, we don't miss PNC operations in the json.

**Review by emadkaramad — APPROVED** [review]

File: packages/message-forwarder/src/forwardMessage/forwardMessage.ts (hunk 1 above)
Comment (chubberlisk) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: Hmmm, you're right Emad - not just for correlation ID as `startBichardProcess` adds the message to S3.

Comment (chubberlisk) [reply packages/message-forwarder/src/forwardMessage/forwardMessage.ts:31]: Thanks for the spot!

**Pull request merged**